src/lib/web-component/filter-keyword/filter-keyword-element.js

```diff
@@ -0,0 +1,41 @@
+class filterKeyword extends HTMLElement {
+  constructor() {
+    super();
+
+    const shadow = this.attachShadow({ mode: 'open' });
+    const filterContainer = document
+      .querySelector('#filterKeywordTemplate')
+      .content.cloneNode(true);
+
+    this.inputBox = filterContainer.querySelector('input[name="keyword"]');
+
+    shadow.appendChild(filterContainer);
+  }
+
+  connectedCallback() {
+    this.inputBoxListener = this.delay(
+      (event) => this.onKeywordSubmit(event.originalTarget.value),
```

Comment:
Use `event.target` instead of the non-standard originalTarget.

Reply:
done.